app/src/main/java/com/hedvig/app/feature/marketing/MarketingViewModel.kt

```diff
@@ -167,37 +119,17 @@ class MarketingViewModel(
   fun onClickLogIn() {
     hAnalytics.buttonClickMarketingLogin()
   }
-
-  companion object {
-    private val PICK_MARKET_INITIAL = PickMarket(
-      isLoading = true,
-      isValid = false,
-      market = null,
-      language = null,
-      availableMarkets = emptyList(),
-    )
-  }
 }
 
-sealed interface ViewState
-
-object Loading : ViewState
-
-data class PickMarket(
-  val isLoading: Boolean,
-  val isValid: Boolean,
-  val market: Market?,
-  val language: Language?,
-  val availableMarkets: List<Market>,
-) : ViewState
-
-sealed interface MarketPicked : ViewState {
-  object Loading : MarketPicked
-
-  data class Loaded(
-    val selectedMarket: Market,
-    val loginMethod: LoginMethod,
-  ) : MarketPicked
+data class ViewState(
```

Comment:
Since this class is outside of the namespace of `MarketingViewModel` (which I agree with, I like them as top level) I think it's a good idea to name it `MarketingViewState`|`MarketingUiState` instead. Otherwise this will pollute the namespace whenever writing ViewState anywhere in the project this will be popping up as a suggestion.